core/src/main/scala/tamer/Tamer.scala

```diff
@@ -312,12 +282,18 @@ object Tamer {
         .withGroupId(groupId)
         .withOffsetRetrieval(OffsetRetrieval.Auto(AutoOffsetStrategy.Earliest))
         .withProperties(properties)
+      val transactionalConsumerSettings = consumerSettings.withProperty(ConsumerConfig.ISOLATION_LEVEL_CONFIG, "read_committed")
       val producerSettings = ProducerSettings(brokers)
         .withCloseTimeout(closeTimeout)
         .withProperties(properties)
```

Comment:
Need also `enable.idempotence = true` as a property of the producer

Reply:
True but according to the docs
> If a TransactionalId is configured, enable.idempotence is implied
https://kafka.apache.org/documentation/#producerconfigs_transactional.id
Also a bunch of other stuff are required at broker level and should be added in some docs, ex.gr.
> by default, transactions require a cluster of at least three brokers